.editorconfig

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = space
+insert_final_newline = true
+
+[*.java]
+ij_continuation_indent_size = 4
+ij_java_class_count_to_use_import_on_demand = 999
+ij_java_names_count_to_use_import_on_demand = 999
+indent_size = 4
+max_line_length = 120
+
+[*.xml]
+max_line_length = 160
```

Comment:
Which one is not covered by spotless or checkstyle ... do we really need next tool

Reply:
See spiked line endings. So might be yes.

Besides the broken repository file endings, wildcard imports are not covered: therefor breaking the build.

https://github.com/apache/maven/pull/2300#issuecomment-2869773513

<img width="1341" alt="image" src="https://github.com/user-attachments/assets/a4651591-9c93-408f-b0d8-f080d88b3fd0" />